src/schema/sources.ts

```diff
@@ -972,12 +976,15 @@ export const resolvers: IResolvers<any, Context> = {
             memberPostingRank: sourceRoleRank[memberPostingRole],
             memberInviteRank: sourceRoleRank[memberInviteRole],
           });
+          await repo.insert(source);
           // Add the logged-in user as admin
+          await createSquadWelcomePost(entityManager, source, ctx.userId);
```

Comment:
I think it's better to put it after adding new source member just to keep it logically right.
Create source -> Add member -> Member publishes welcome post

Reply:
Ah, yes, that makes sense. Will update quickly 🚀